core/domain/issue_registry_test.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the issue registry."""
+
+from core.domain import issue_registry
+from core.tests import test_utils
+
+
+class IssueRegistryUnitTests(test_utils.GenericTestBase):
+    """Test for the issue registry."""
+
+    def test_issue_registry(self):
+        """Do some sanity checks on the issue registry."""
+        self.assertEqual(
+            len(issue_registry.Registry.get_all_issues()),
```

Comment:
Seems this test is failing b/c the Issues don't exist yet. You can get around this by passing a "testonly" variable to fill the members under-test directly:

```py
class BaseExplorationIssueSpec:
    pass
class TestIssue(BaseExplorationIssueSpec):
    pass

issue_registry.Registry._refresh({'TestIssue': TestIssue})
self.assertEqual(len(issue_registry.Registry.get_all_issues()), 1)
```

Reply:
Done.